### PR TITLE
[RU]Desu new domain

### LINKED ISSUE #17954: Desu: new domain
### Source information

desu

### Source language

ru

### Source new URL

desu.win

### Other details

_No response_

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by e-shl**
Closes #17954

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by e-shl** [comment]
The links to the images in the API have been updated, now the domain change should work correctly.

**Comment by i3sey** [comment]
Thanks for the fix, I'm waiting for the merge.

**Comment by jeffangelion** [comment]
This PR fixes Desu extension, so it needs to be merged

File: src/ru/desu/AndroidManifest.xml [hunk 1]
```diff
@@ -18,6 +18,10 @@
                     android:host="desu.me"
```
Comment (arkon) [inline src/ru/desu/AndroidManifest.xml:18]: Can this be removed then?

Comment (e-shl) [reply src/ru/desu/AndroidManifest.xml:18]: The domain is not completely closed and may even return some day

**Pull request merged**